codemods/src/6.0.0-7.0.0/__testfixtures__/replace-nudge.output.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Nudge, Pulsar } from 'evergreen-ui';
+
+const props = {
+    isShown: true,
+    onClick: () => {}
+}
+
+const Component = () => {
+  return (
+    <React.Fragment>
+      <Pulsar></Pulsar>
+      <Pulsar />
+      <Pulsar position='top-left' />
+      <Pulsar onClick={() => {}} />
```

Comment:
Upon further inspection, this codemod might not be as useful in its current state. The `Pulsar` does not wrap children like its predecessor, so to achieve the same visual effect of

```tsx
<Nudge
  position={Position.TOP_LEFT}
  isShown={true}
>
  <Description />
</Nudge>
```
<img width="572" alt="image" src="https://user-images.githubusercontent.com/11774799/205506610-a4f4d376-b1bb-421a-938a-555cfd505f71.png">

You'd need to make a wrapping `Pane` with `position="relative"` (based on the examples on the [doc site](https://evergreen.segment.com/components/pulsar))
```tsx
<Pane position="relative">
  <Pulsar position={Position.TOP_LEFT} />
  <Description />
</Pane>
```
<img width="561" alt="image" src="https://user-images.githubusercontent.com/11774799/205506655-29c554d2-09b8-4c6d-8fef-2479d23d8b10.png">

I might take another stab at this to make it a little smarter. 

We also might be able to programmatically swap the `tooltipContent` with a `<Tooltip>`, but it will have a slightly different look. I think the `Pulsar` needs to forward its ref first, right this results in a console error and no Tooltip being rendered on hover:

```tsx
<Tooltip content="Example tooltip content">
  <Pulsar position={Position.TOP_LEFT} />
</Tooltip>
```

```
Warning: Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?

Check the render method of `Stack`.
    at eval (https://pvtkqq.csb.app/node_modules/evergreen-ui/esm/pulsar/src/Pulsar.js:53:30)
    at Stack (https://pvtkqq.csb.app/node_modules/evergreen-ui/esm/stack/src/Stack.js:13:23)
    at Positioner (https://pvtkqq.csb.app/node_modules/evergreen-ui/esm/positioner/src/Positioner.js:50:22)
    at Tooltip (https://pvtkqq.csb.app/node_modules/evergreen-ui/esm/tooltip/src/Tooltip.js:60:33)
    at div
    at eval (https://pvtkqq.csb.app/node_modules/ui-box/dist/src/box.js:21:11)
    at Pane (https://pvtkqq.csb.app/node_modules/evergreen-ui/esm/layers/src/Pane.js:57:31)
    at div
    at eval (https://pvtkqq.csb.app/node_modules/ui-box/dist/src/box.js:21:11)
    at Pane (https://pvtkqq.csb.app/node_modules/evergreen-ui/esm/layers/src/Pane.js:57:31)
    at App
 ```

Reply:
This has been fixed with [`0dd4b9d` (#1558)](https://github.com/segmentio/evergreen/pull/1558/commits/0dd4b9d568a64314467f1d1be419445f9ed46c91) and #1565